modules/javafx.base/src/test/java/test/JUnit5Test.java

```diff
@@ -39,4 +43,30 @@ void junit5ShouldWork() {
         assertNotNull(this);
         System.err.println("JUnit 5 test working!");
     }
+
+    static int callCount;
+    static int[] intValues = {1, 2, 3};
+    @ValueSource(ints = {1, 2, 3})
```

Comment:
Minor: maybe add a blank line before this line?

Reply:
Added new line.